apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/V1/SessionPoolTests.TargetedSessionPoolTests.cs

```diff
@@ -401,13 +400,13 @@ public async Task WaitForPoolAsync_Normal()
                     // We allow 20 session creates at a time, and each takes 5 seconds.
                     // We'll need at least 6 read/write sessions too, but those will be taken from the first 20
                     // as we err on the side of over-creating read/write transactions. (When the first 20
-                    // all return, they'll all notice we have no transactions.) That won't slow down the
+                    // all return, they'll probably all notice we have no transactions.) That won't slow down the
                     // second set of creations.
                     var timeAfter = client.Clock.GetCurrentDateTimeUtc();
                     Assert.Equal(TimeSpan.FromSeconds(10), timeAfter - timeBefore);
```

Comment:
This is the only test that is failing in Travis now because:
Assert.Equal() Failure
Expected: 00:00:10
Actual:   00:00:11
 I know it's weird, but that probably means that the 100ms loop is almost stable, and that in some cases, one of the begin transaction delays is coming in after we have moved the clock 5 seconds two times. If it happens with one of the begin transactions delays, it might happen also with one of the second batch creation delays, so to be on the safe side, I think I'd put this as:
`Assert.Equal(timeAfter - timeBefore, TimeSpan.FromSeconds(10), TimeSpan.FromSeconds(16));`

That or increase `IdleTimeBeforeAdvancing` a little to make "sure" that everything comes in before moving the clock.